src/transformState.js

```diff
@@ -5,7 +5,19 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    if (action.type === 'addProperties') {
+      Object.assign(state, action.extraData);
+    } else if ((action.type === 'removeProperties')) {
+      for (const removedKey of action.keysToRemove) {
```

Comment:
```suggestion
      for (const keyToRemove of action.keysToRemove) {
```
it's good practice to name such values in this way during mapping, schematically: `for (const singular of plural)`